package.json

```diff
@@ -14,6 +14,161 @@
+    "test:e2e.verbose": "npm run build:umd && npm run build:languages && npm run test:e2e.dump && npm run test:e2e.puppeteer -- --verbose",
+    "test:e2e.watch": "node ./.config/bin/trigger-on-stdout-change.js \"concurrently --raw --kill-others \\\"npm run build:umd -- --watch\\\" \\\"npm run test:e2e.dump -- --watch\\\"\" \"npm run test:e2e.puppeteer\"",
+    "test:e2e.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-e2e env-cmd -f ./hot.config.js webpack --hide-modules helpers=./test/helpers/index.js e2e=./test/e2e/index.js",
+    "test:e2e.puppeteer": "node test/scripts/run-puppeteer.js test/E2ERunner.html",
+    "test:unit": "cross-env-shell npm_config_testPathPattern=. BABEL_ENV=commonjs env-cmd -f ./hot.config.js jest --testPathPattern=$npm_config_testPathPattern",
+    "test:unit.watch": "cross-env-shell BABEL_ENV=commonjs env-cmd -f ./hot.config.js jest --testPathPattern=$npm_config_testPathPattern --watch",
+    "test:mobile.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-mobile env-cmd --no-override -f ./hot.config.js webpack --hide-modules helpers=./test/helpers/index.js mobile=./test/e2e/mobile/index.js",
+    "test:types": "tsc -p ./test/types",
+    "test:ci": "node scripts/run-targeted-tests.mjs",
+    "test:all": "npm run in handsontable build && npm run all test",
+    "watch": "cross-env-shell BABEL_ENV=commonjs NODE_ENV=watch env-cmd -f ./hot.config.js webpack --hide-modules --watch src/index.js",
+    "build": "npm run clean && npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd.min && npm run build:languages && npm run build:languages.es && npm run build:languages.min",
+    "build:commonjs": "cross-env-shell BABEL_ENV=commonjs_dist env-cmd -f ./hot.config.js babel src --out-dir tmp",
+    "build:es": "cross-env-shell BABEL_ENV=es env-cmd -f ./hot.config.js babel src --out-file-extension .mjs --out-dir tmp",
+    "build:umd": "cross-env-shell BABEL_ENV=commonjs NODE_ENV=development env-cmd -f ./hot.config.js webpack --hide-modules ./src/index.js",
+    "build:umd.min": "cross-env-shell BABEL_ENV=commonjs NODE_ENV=production env-cmd -f ./hot.config.js webpack --hide-modules ./src/index.js",
+    "build:walkontable": "cross-env-shell BABEL_ENV=commonjs NODE_ENV=walkontable env-cmd -f ./hot.config.js webpack --hide-modules ./src/3rdparty/walkontable/src/index.js",
+    "build:languages": "cross-env-shell BABEL_ENV=commonjs NODE_ENV=languages-development env-cmd -f ./hot.config.js webpack",
+    "build:languages.es": "cross-env-shell BABEL_ENV=es_languages babel src/i18n/languages --out-file-extension .mjs --out-dir languages",
+    "build:languages.min": "cross-env-shell BABEL_ENV=commonjs NODE_ENV=languages-production env-cmd -f ./hot.config.js webpack",
+    "freeze": "node scripts/freeze.mjs",
+    "release": "node scripts/release.mjs",
+    "postbuild": "node ./scripts/prepare-package-for-publish",
```

Comment:
I know that this request is out of the PR scope. But, can I ask you to rewrite the `prepare-package-for-publish` script to `msj`? Like you did for the rest of the scripts. 

This will keep all scripts consistent.

Reply:
Sure dedae71